src/Nethermind/Nethermind.Runner/Ethereum/Steps/DatabaseMigrations.cs

```diff
@@ -152,6 +152,12 @@ private long MinBlockNumber
 
         private void RunBloomMigration(CancellationToken token)
         {
+            BlockHeader GetMissingBLockHeader(long i)
+            {
+                if (_logger.IsWarn) _logger.Warn(GetLogMessage("warning", $"Header for block {i} not found. Database might be corrupted and BloomDb will not be correct."));
```

Comment:
Should we blow up migration?

Reply:
no, you will just be missing some blocks. I would even change the warning to say - there will be no blocks for this header